website/src/index.html

```diff
@@ -0,0 +1,210 @@
+                    <p class="marketing__desc js-reveal">
+                        It is highly customizable, not only on the look and feel (with different month and day visions,
+                        different themes, and more), but also offers lots of functionalities such as defining how many
+                        hours
+                        you
+                        usually work per day, which week days, allow you to quickly source holidays based on your region
+                        (Country, State, and City), and much more.
+                    </p>
+
+                    <div class="marketing__features js-reveal">
+                        <div class="marketing__feature">
+                            <img class="marketing__feature__img" src="images/theme.svg" alt="Time To Leave Logo">
+                            <h3 class="marketing__feature__title">Themes</h3>
+                            <p class="marketing__feature__desc">Time to Leave comes with three themes right out the box.
+                                Light, Dark, and are new Cadent
+                                Start theme. </p>
+                        </div>
+                        <div class="marketing__feature">
+                            <img class="marketing__feature__img" src="images/daymonth.svg" alt="Time To Leave Logo">
+                            <h3 class="marketing__feature__title">Views</h3>
+                            <p class="marketing__feature__desc">Pick the between Day and Month log your work time;
+                                lunch, start and end of your day.</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div class="customization__illustration">
+                <img class="js-customization-image--1" src="./images/dayView.png" alt="Time to Leave interface">
+                <img class="js-customization-image--2" src="./images/darkDayView.png" alt="Time to Leave interface darkmode">
+                <img class="js-customization-image--3" src="./images/lightDayView.png" alt="Time to Leave interface">
+            </div>
+        </section>
+
+
+        <section class="notification">
+            <div class="notification__illustration js-notification--trigger">
+                <img class="js-notification-image--1" src="./images/28jz6fm6.png" alt="Time to leave preferences interface">
+                <img class="js-notification-image--2" src="./images/vgmvfdx0.png" alt="Time to leave notification">
+            </div>
+
+            <div class="marketing">
+                <div class="marketing__container js-reveal">
+                    <h2 class="marketing__title">Get Notify When It's Time</h2>
+                    <p class="marketing__desc">Log the time you started working today, and the program will compute what
+                        time you should leave the
+                        office, considering the time you spent on lunch. </p>
```

Comment:
```suggestion
                    <p class="marketing__desc">Log the time you started working today, and the program will compute what
                        time you should leave the office, considering the time you spent on breaks, such as lunch. </p>
```
With the newest option there can be more breaks than just the lunch